### PR TITLE
Primer commit

### PR DESCRIPTION
**Pull request by xWylo16**

### CODE REVIEW

File: src/app/modules/about/about.page.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
       <ion-menu-button></ion-menu-button>
     </ion-buttons>
     <ion-title>
-      About
+      About xd jsjsjsj s
```
Comment (Bearnal19) [inline src/app/modules/about/about.page.html]: Delete jsjs

**Review by Bearnal19 — CHANGES_REQUESTED** [review]
Make proper changes

File: src/app/modules/about/about.page.html (hunk 1 above)
Comment (xWylo16) [reply src/app/modules/about/about.page.html]: done

**Review by Bearnal19 — APPROVED** [review]
lgtm

**Pull request merged**